interpreter/cling/tools/packaging/cpt.py

```diff
@@ -1975,6 +1975,7 @@ def custom_input(prompt, always_yes=False):
     tar_required = True
 
 if args['check_requirements']:
+    args['skip_cleanup'] = True
```

Comment:
Why do we need the `skip-cleanup` when checking for the requirements?

Reply:
@vgvassilev my bad I think I misread your email. I assumed you meant add skip-cleanup to the check_requirements flag too.

Reply:
Ah, no, only to the other one.

Reply:
Yeah just removed @vgvassilev my bad